airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -4,7 +4,7 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Postgres Source Spec",
     "type": "object",
-    "required": ["host", "password", "port", "database", "username"],
```

Comment:
password isn't a required field in postgres. furthermore in practice, this makes it inconvenient to use most local databases.